File: server-scala-play-server/app/model/NodeRequest.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for NodeRequest.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class NodeRequest(
  x: Option[Int],
  y: Option[Int],
  z: Option[Int],
  latitude: Option[Double],
  longitude: Option[Double],
  altitude: Option[Double],
  date: Option[Long],
  isWaypoint: Option[Boolean],
  accountId: Option[Long],
  node: Option[Node]
)

object NodeRequest {
  implicit lazy val nodeRequestJsonFormat: Format[NodeRequest] = Json.format[NodeRequest]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/NodeRequest.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class NodeRequest 
{
    public int X { get; set; }
    public int Y { get; set; }
    public int Z { get; set; }
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public double Altitude { get; set; }
    public long Date { get; set; }
    public bool IsWaypoint { get; set; }
    public long AccountId { get; set; }
    public Node Node { get; set; }
}


